src/mz/src/ui.rs

```diff
@@ -65,6 +65,17 @@ impl OutputFormatter {
         }
     }
 
+    /// Outputs a prefix warning.
+    pub fn output_warning(&self, msg: &str) -> Result<(), Error> {
+        if self.no_color {
+            eprintln!("\n* Warning * {}", msg);
+        } else {
+            eprintln!("\n\x1b[93m* Warning * \x1b[0m {}", msg);
```

Comment:
Hardcoding ASCII terminal codes like this is not ideal. Best practice would be to use one of the terminal coloring libraries for Rust. Testdrive uses `termcolor`, for example.